Rakefile

```diff
@@ -8,7 +8,14 @@ require "hanami/devtools/rake_tasks"
 namespace :spec do
   RSpec::Core::RakeTask.new(:unit) do |task|
     file_list = FileList["spec/**/*_spec.rb"]
-    file_list = file_list.exclude("spec/{integration,isolation}/**/*_spec.rb")
+    file_list = file_list.exclude("spec/{integration}/**/*_spec.rb")
```

Comment:
I think you forgot to exclude  isolation tests 🤔

Reply:
That was intentional as there aren't any isolation tests, or an `spec/isolation` directory

Reply:
@landongrindheim I think this isn't a required change, as it doesn't affect the outcome of unit tests. The code will be ready to accept isolation specs, if needed.